silo-vaults/contracts/incentives/VaultIncentivesModule.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.28;
+
+import {Ownable2Step, Ownable} from "openzeppelin5/access/Ownable2Step.sol";
+import {EnumerableSet} from "openzeppelin5/utils/structs/EnumerableSet.sol";
+
+import {IVaultIncentivesModule} from "../interfaces/IVaultIncentivesModule.sol";
+import {IIncentivesClaimingLogic} from "../interfaces/IIncentivesClaimingLogic.sol";
+import {IIncentivesDistributionSolution} from "../interfaces/IIncentivesDistributionSolution.sol";
+
+/// @title Vault Incentives Module
+contract VaultIncentivesModule is IVaultIncentivesModule, Ownable2Step {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    bytes32 public constant INCENTIVES_CLAIMING_LOGIC_PING = keccak256(abi.encode("IncentivesClaimingLogic"));
```

Comment:
I'm against the ping method. We have custom interface that is being implemented and it serves as "ping" itself.

Reply:
Removed.